scripts/sysman/rasState.yml

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for RAS Get State and Clear State"
+version: "1.7"
+--- #--------------------------------------------------------------------------
+type: macro
+desc: "RAS Get State Extension Name"
+version: "1.7"
+name: $S_RAS_GET_STATE_EXP_NAME
+value: '"$XS_extension_ras_state"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "RAS Get State Extension Version(s)"
+version: "1.7"
+name: $s_ras_state_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension structure for providing RAS error counters for different error sets"
+version: "1.7"
+class: $sRas
+name: $s_ras_state_exp_t
+members:
+    - type: $s_ras_error_cat_t
+      name: "category"
+      desc: "[out] category for which error counter is provided."
+    - type: uint64_t
+      name: errorCounter
+      desc: "[out] Current value of RAS counter for specific error category."
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Ras Get State"
+class: $sRas
+name: RasGetStateExp
```

Comment:
why double 
Ras? in the class and in the name?

Reply:
oops, fixed both here and for clear state